climada/hazard/base.py

```diff
@@ -1393,3 +1442,167 @@ def _read_att_excel(self, file_name, var_names):
         self.intensity = sparse.csr_matrix(dfr.values[:, 1:num_events + 1].transpose())
         self.fraction = sparse.csr_matrix(np.ones(self.intensity.shape,
                                                   dtype=np.float))
+
+    def concatenate(self, haz_src, append=False, centroids=None):
+        """
+        Concatenate events of several hazards of same type.
+
+        This is a wrapper method for hazard.base.concatenate_hazard
+        to ensure backwards compatibility.
+
+        Parameters
+        ----------
+        haz_src: list(climada.hazard.Hazard())
+            Hazard instances of the same hazard type
+        append : bool, optional
+            If True, append the haz_src to current object. If False,
+            replace all attributes from current objects by concatenated
+            hazards from haz_src. The default is False.
+        centtroids: climada.hazard.Centroids(), optional
+            Centroids instance on which to map the concatenated hazard.
+            Default is None.
+
+        Returns
+        -------
+        None.
+
+        """
+        if append:
+            haz_src = [self] + haz_src
+        self.__dict__ = copy.deepcopy(
+            concatenate_hazard(haz_src, centroids).__dict__
+            )
+
+
+def join_centroids(haz_list):
+    """
+    Find centroids comon to all.
```

Comment:
"common to all" implies set intersection
```suggestion
    Create the union of centroids from the inputs.
```